LineApp/src/main/java/org/example/models/PullRequest.java

```diff
@@ -0,0 +1,210 @@
+package org.example.models;
+
+import jakarta.persistence.*;
+
+import java.io.Serial;
+import java.time.ZonedDateTime;
+
+@Entity
+@Table(name = "PullRequest")
+public class PullRequest extends Model{
+    @Serial
+    private static final long serialVersionUID = 3460490108006801863L;
+
+    @Column(name = "repo", nullable = false)
+    private String repo;
+    @Column(name = "base", nullable = false)
+    private String base;
+    @Column(name = "number", nullable = false)
+    private int number;
+    @Column(name = "author_id", nullable = false)
+    private int author_id;
+    @Column(name = "author", nullable = false)
+    private String author;
+    @Column(name = "created_at", nullable = false)
+    private ZonedDateTime created_at;
```

Comment:
here are some general recommendations about naming in Java:

Use camel case for most variable names, method names, and parameter names. This is the most widely accepted convention in Java and is used in the Java Standard Library and many third-party libraries.

Use pascal case for class names, interface names, and enum types. This aligns with Java's class naming convention and makes it clear that you're dealing with a type definition.

Avoid using snake case for identifiers in standard Java code. While it's not common in Java itself, it may be used in specific contexts such as configuration files or when interacting with systems or libraries that use snake case.

Hint: you use snake case here, which is generally not welcomed